releases.json

```diff
@@ -1563,6 +1563,14 @@
       "2.5.0-1"
     ]
   },
+  "tree-sitter": {
+    "dependency_names": [
+      "tree-sitter"
+    ],
+    "versions": [
+      "0.15.7"
```

Comment:
All versions in this file must end in a hyphen followed by an integer indicating the WrapDB revision, starting at 1.